src/test/java/io/getstream/chat/java/ChannelTypeTest.java

```diff
@@ -132,4 +133,26 @@ void whenCreatingDefaultChannelType_thenCanUpdateWithNoException() {
     Assertions.assertDoesNotThrow(
         () -> ChannelType.update(channelName).automod(AutoMod.SIMPLE).request());
   }
+
+  @DisplayName("Can create channel type with specific grants")
+  @Test
+  void whenManipulatingChannelTypeWithGrants_throwsNoException() {
+    String channelTypeName = RandomStringUtils.randomAlphabetic(10);
+    var expectedGrants = List.of("read-channel", "create-message");
+    var channelGrants = new HashMap<String, List<String>>();
+    channelGrants.put("channel_member", expectedGrants);
+    Assertions.assertDoesNotThrow(
+        () ->
+            ChannelType.create()
+                .withDefaultConfig()
+                .grants(channelGrants)
+                .name(channelTypeName)
+                .request());
+    pause();
```

Comment:
There is new method `wait`. To wait until some condition has been met.

Reply:
Done @driver-devel thanks! 🚀